Assets/MixedRealityToolkit/Utilities/WindowsApiChecker.cs

```diff
@@ -33,37 +36,64 @@ public static void CheckApiContracts()
             UniversalApiContractV5_IsAvailable = false;
             UniversalApiContractV4_IsAvailable = false;
             UniversalApiContractV3_IsAvailable = false;
-#endif
+#endif // WINDOWS_UWP
+#pragma warning restore 0618
+        }
+
+        /// <summary>
+        /// Checks to see if the requested method is present on the current platform.
+        /// </summary>
+        /// <param name="namespaceName">The namespace (ex: "Windows.UI.Input.Spatial") containing the class.</param>
+        /// <param name="className">The name of the class containing the method (ex: "SpatialInteractionMananger").</param>
+        /// <param name="methodName">The name of the method (ex: "IsSourceKindSupported").</param>
+        /// <returns>True if the method is available and can be called. Otherwise, false.</returns>
+        public static bool IsMethodAvailable(
```

Comment:
We have some similar API calls in MRTK already, like 
https://github.com/microsoft/MixedRealityToolkit-Unity/blob/f060a82bb1c0c9286e372982fdc640ffe6592107/Assets/MixedRealityToolkit.Providers/WindowsMixedReality/XR2018/WindowsMixedRealityEyeGazeDataProvider.cs#L65

Doesn't have to be part of this PR, but could be worth adding the `Property` and `Type` versions of this API as well

Reply:
Issue #7426